### PR TITLE
Remove Travis and use Github Actions

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+env:
+  DOTNET_VERSION: 3.0.100
+
+jobs:
+
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Chectout
```
Comment (shargon) [inline .github/workflows/dotnetcore.yml]:
```suggestion
    - name: Checkout
```

Comment (vncoelho) [reply .github/workflows/dotnetcore.yml]: I also noticed this on the NEO, we can fix together with another PR there.

**Comment by erikzhang** [comment]
We can automatically publish the release if the version changed.

**Comment by lock9** [comment]
Hmm, not sure if I understand. I can't see from the current configuration release instructions. Are you sure the release process is automated? Where can I find the rules?

**Comment by erikzhang** [comment]
@lock I need to do some research first. I don't know how to implement it. We need to improve it later.

**Pull request merged**